integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r3/TaskFhirResourceIntegrationTest.java

```diff
@@ -169,6 +176,45 @@ public void shouldCreateNewTaskAsJson() throws Exception {
 		assertThat(newTask.getIntent(), equalTo(task.getIntent()));
 		assertThat(task.getAuthoredOn(), equalTo(task.getAuthoredOn()));
 		assertThat(task.getLastModified(), equalTo(task.getLastModified()));
+		assertThat(newTask.getOutput(), hasSize(4));
+		assertThat(newTask.getInput(), hasSize(4));
+		for (Task.TaskOutputComponent output : newTask.getOutput()) {
+			if (output.getValue() instanceof Reference) {
+				assertThat(((Reference) output.getValue()).getReference(),
+				    equalTo("DiagnosticReport/9b6f11dd-55d2-4ff6-8ec2-73f6ad1b759e"));
+				
+			} else if (output.getValue() instanceof DateTimeType) {
+				DateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
There are some examples of how we can test dates. Please look at them... I don't love converting them to strings, etc., because this can make the time-stamps locale specific.

Reply:
makes sense